vllm/model_executor/layers/quantization/compressed_tensors/utils.py

```diff
@@ -209,6 +209,23 @@ def _find_first_match(value: str,
     return None
 
 
+def get_compressed_tensors_cache_scale(name: str) -> Optional[str]:
+    """
+    Check whether the param name matches the format for k/v cache scales
+    in compressed-tensors. If this is the case, return its equivalent
+    param name expected by vLLM
+
+    :param name: param name
+    :return: matching param name for KV cache scale in vLLM
+    """
+    if name.endswith(".output_scale") and ".k_proj" in name:
+        return name.replace(".k_proj.output_scale", ".attn.k_scale")
+    elif name.endswith(".output_scale") and ".v_proj" in name:
```

Comment:
nit: Use `if` when each if-branch returns.
```suggestion
    if name.endswith(".output_scale") and ".v_proj" in name:
```